docs/guides/migrating-react-router-app.md

```diff
@@ -184,6 +184,8 @@ Inside of your `$.jsx` file, all we need to do is export the code from our old r
 export { default } from "~/old-app/app";
 ```
 
+<docs-warning><strong>Important:</strong> make sure `app/routes/index.[jsx|tsx]` exists in your Remix app. Remix will not automatically render your `/` React Router fallback route, which means a blank page will be rendered without the index file in the `routes` directory.</docs-warning>
+
```

Comment:
Instead of using a warning here, could we just change the section to be "Create an Index and a Catch-all route" and export your old app (i.e., `<Routes>` tree) from both?